packages/pagination/src/FirstPage.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '../../button/src/index.js';
+import { i18n } from '@lingui/core';
+import { pagination as ccPagination } from '@warp-ds/css/component-classes';
+import { usePagination } from './PaginationContainer.js';
+import IconChevronDoubleLeft16 from '@warp-ds/icons/react/chevron-double-left-16';
+import React from 'react';
+
+type FirstPageProps = {
+  /**
+   * @default First page
+   */
+  'aria-label'?: string;
```

Comment:
I think you should use `aria-label` or the translated default in line 37

Reply:
What do you mean? For someone using the FirstPage.tsx, they can either suppy their own aria-label or use the one we have in this library

Reply:
I meant that you should maybe do what you did in the PaginationContainer here: https://github.com/warp-ds/react/pull/189/files#diff-b4a4e56035a870c1373c2bf17afc691e9fdd9321e42da35f7a913d36ef3cbf71R67-R74
This way if aria-label prop exists, it would be used instead of the default one.